lib/static/css/main.css

```diff
@@ -841,7 +841,14 @@ input[type='search'] {
   position: absolute;
   right: 10px;
   top: 15px;
+  border-radius: 3.5px;
+  border: solid 1px #97979769;
+  margin: 36px 0 -36px 0;
 }
+input#search_input_react {
+  padding-left: 10px!important;
```

Comment:
I will always ask about `!important`, I believe you should avoid it at all costs and will try to figure out if the reason it is needed is because of css specificity instead of simply adding the important overlord because once you add one, the only way to override it is to use it again.
Having said all of that, why is it needed here? Can you make it work without it?